main.go

```diff
@@ -129,10 +129,6 @@ func tr1d1um(arguments []string) (exitCode int) {
 
 	r := mux.NewRouter()
 
-	r.NotFoundHandler = http.HandlerFunc(func(w http.ResponseWriter, _ *http.Request) {
```

Comment:
Reverting to original handling of unmatched routes. This will help to stay consistent throughout the different ways the API can be exercised (paths, method, etc.)